src/main/java/core/basesyntax/Text.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Text {
```

Comment:
```suggestion
public interface Drawable {
```
interface describes behavior, Drawable means can be drawn